tools/shared.py

```diff
@@ -895,10 +895,10 @@ def get_vanilla_file():
       logging.debug('failed to use vanilla file, will re-check: ' + str(e))
       is_vanilla = check_vanilla()
     temp_cache = None
+    os.environ['EMCC_WASM_BACKEND'] = str(is_vanilla)
```

Comment:
Ah sorry, I mean this one: https://github.com/kripken/emscripten/pull/4876/files#diff-b97059c38de7432017d59dee3e87bc72R1261